src/Leaderboard.css

```diff
@@ -0,0 +1,63 @@
+/* Color variables */
+:root {
+  --main-green: #076B49;
+  --sec-green: #74CC67;
+  --white: #FFF;
+  --lb-gray: #E0E9E6;
+}
+
```

Comment:
This is good CSS however each of these classes will go to the global css DOM and can have clashes possbly later. It would be best to use .lb-container for each subclassed html example below
.lb-container .lb-user-stats {
  width: 80%;
  height: 5vh;
  display: flex;
  align-items: center;
  justify-content: space-around;
  border: 0.2rem var(--main-green) solid;
  border-radius: 5rem;
  margin-bottom: 2rem;
}

Reply:
same to the other classses, might need to add the other parent classes and yes it is somewhat redundent code but it is only in this file and does not need to be comiled unline SASS